contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViewConstants.java

```diff
@@ -488,7 +546,7 @@
           View.Name.create("grpc.io/client/started_count/minute"),
           "Minute stats on the number of client RPCs started",
           RPC_CLIENT_STARTED_COUNT,
-          MEAN,
+          COUNT,
```

Comment:
Are there any backwards compatibility issues with the changes to existing views?

Reply:
Not in our library or gPRC, and it's unlikely to break user code unless they are doing a casting on AggreagtionData of these Views. Do you think we should deprecate the old views instead?

Reply:
I was mainly concerned about gRPC, since I think that they are the only direct user of the RPC constants.  Since it doesn't affect gRPC, I think this is fine.

Reply:
SGTM